submitter/jobsubmitter.py

```diff
@@ -10,7 +10,7 @@ class App(object):
 
     def factory(app):
         if app.get('github'):
-            repo = app['github']['repository']
+            repo = app['github']['repository'].replace(".git", "")
```

Comment:
Why do we need to replace .git?

Reply:
In order to use the SSH key we're passing in `GIT_SSH_COMMAND`, we have to pull using the SSH url: `git@github.com:mskcc/ACCESS-Pipeline.git`